innopoints/models/activity.py

```diff
@@ -74,6 +80,9 @@ def dates(self):
     @property
     def vacant_spots(self):
         """Return the amount of vacant spots for the activity"""
+        if self.people_required is None:
+            return -1
+
         accepted = Application.query.filter_by(activity_id=self.id,
                                                status=ApplicationStatus.approved).count()
         return max(self.people_required - accepted, -1)
```

Comment:
No longer need the `max` because the `-1` case is handled above

Reply:
Just making sure if in any case there will be more accepted applications. Doesn't hurt